src/soso/strategies/spase.py

```diff
@@ -41,22 +41,42 @@ def __init__(self, file: str, **kwargs: dict):
         super().__init__(metadata=etree.parse(file))
         self.file = file
         self.schema_version = get_schema_version(self.metadata)
+        self.namespaces = {"spase": "http://www.spase-group.org/data/schema"}
         self.kwargs = kwargs
 
     def get_id(self) -> None:
```

Comment:
Update the return type from `None` to `str` to match the result. If the return value can be `None`, because the SPASE element is not required, then define the return type as `Union[str, None]`